app/javascript/packages/password-toggle/package.json

```diff
@@ -4,10 +4,5 @@
   "version": "1.0.0",
   "peerDependencies": {
     "react": "^17.0.2"
```

Comment:
Related to previous comment, I think the changes here are valid, but we'll to remove this `peerDependencies` as well.

Reply:
This comment was marked resolved, but I think it's still valid (changes needed).